use-cases/index.mdx

```diff
@@ -0,0 +1,10 @@
+---
+title: Uses - XRPL.org
+---
```

Comment:
I suggest moving `history/index.md` to `about/history.mdx` and `use-cases/index.mdx` to `about/use-cases.mdx`. This should result in the files being in the appropriate paths for the new nav as well as the file structure of the repo.

Also, you can put a redirect from the old URL into the frontmatter here, like this:

```suggestion
---
title: Uses - XRPL.org
redirectFrom:
  - /uses.html
---
```